--- cy_ethif.c
+++ cy_ethif.c
@@ -62,7 +62,7 @@
 static CEDI_Statistics  * cyp_ethif_statistic[CY_ETH_DEFINE_NUM_IP];
 
 /** Private data memory allocation  */
-static uint8_t cy_ethif_privatedata[CY_ETH_DEFINE_NUM_IP][1800] = {{0,},};
+/*static*/ uint8_t cy_ethif_privatedata[CY_ETH_DEFINE_NUM_IP][1800] = {{0,},};
 
 /** Tx descriptors */
 /** CY_ETH_DEFINE_NUM_TXQS * CY_ETH_DEFINE_NUM_IP to avoid race condition in hardware; each BD list for particular queue must have 1 more BD */
